llm/ggml.go

```diff
@@ -1,16 +1,14 @@
 package llm
```

Comment:
Is this the right filename for this?

Lots of changes here, is this from merging `gguf.go`?